setup.py

```diff
@@ -7,46 +7,45 @@
 import re
 from setuptools import setup
 
-with open('flask_meld/__init__.py', 'r') as f:
-    version = re.search(r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]',
-                        f.read(), re.MULTILINE).group(1)
+with open("flask_meld/__init__.py", "r") as f:
+    version = re.search(
+        r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]', f.read(), re.MULTILINE
```

Comment:
seems complex, what about `[line.split(" = ")[1] for line in f.readlines() if line.startswith("__version__")][0]`?

I have not used this yet, but might be worth looking into: https://github.com/python-versioneer/python-versioneer